docs/pages/guides/using-graphql.md

```diff
@@ -324,7 +324,7 @@ class CreatePage extends React.Component {
             disabled={!this.state.text || !this.state.title}
             type="submit"
             value="Create"
-          />{' '}
```

Comment:
this is actually necessary in some cases (https://stackoverflow.com/questions/38547558/why-do-people-put-in-their-react-jsx), can I ask why you want it removed?

Reply:
yes it's necessary in some cases  but it' not a good idea to add space like this you can do this 

```{` or cancel`}``` inside <a>

Reply:
okay, could you change that in your PR?

Reply:
sure i can